lib/stateView/document/query/errors/NotIndexedFieldError.js

```diff
@@ -0,0 +1,24 @@
+const ValidationError = require('./ValidationError');
+
+class NotIndexedFieldError extends ValidationError {
+  /**
+   *
+   * @param {string} field
+   */
+  constructor(field) {
```

Comment:
You know, I would provide not indexed field rather than indexed in error.

Reply:
It's already not indexed.

Reply:
Then we need to rename the getter.